spacemacs/files.vim

```diff
@@ -1,50 +1,65 @@
 " Show bookmarks.
 " Create bookmark 0 with <C-S-0>, Create bookmark 1 with <C-S-1> and so on.
 " Go to bookmark 0 with <C-0>, go to bookmark 1 with <C-1> and so on.
+let g:WhichKeyDesc_Files_ShowBookmarks = "<leader>fb show-bookmarks"
 nnoremap <leader>fb    :action ShowBookmarks<CR>
 vnoremap <leader>fb    :action ShowBookmarks<CR>
 
+""" IDE Submenu ---------------------------------
+let g:WhichKeyDesc_Files_IDE = "<leader>fe +IDE"
```

Comment:
`+IDE` seemed a good fit since the `E` in `IDE` matches the `e` in `<leader>fe`.  But, not 100% sure if that name captures the intent of this menu...what do you think?